js/index.js

```diff
@@ -239,6 +239,38 @@ onload = function() {
   Inspector.init();
   Inspector.showPane('pane-info');
 
+  /** Loads a `File` object as a package.json. */
+  async function loadPackageJSON(file) {
```

Comment:
I believe this function can be pulled out to top level of this file. (It doesn't reference any graph()-scoped vars here, does it?)  If so, just move to the end of this file.

(And, yes, I know this codebase is a bit of a mess overall so I probably shouldn't get too picky.  This whole thing was a weekend-project that I've never really gotten around to cleaning up.  But... what's a guy to do?  :-) )

Reply:
Agreed -- fixed.